xs2a-adapter-aspsp-registry/src/test/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepositoryTest.java

```diff
@@ -136,8 +161,8 @@ public void findAllWithPagination() {
         Aspsp aspsp3 = new Aspsp();
         luceneAspspRepository.save(aspsp3);
 
-        List<Aspsp> firstPage = luceneAspspRepository.findAll(2);
-        assertThat(firstPage).hasSize(2);
+        List<Aspsp> firstPage = luceneAspspRepository.findAll(3);
```

Comment:
Why is this changed? The tests inserts 3 records and then verifies that with page size = 2 there are 2 records returned on the first page and one on the second.

Reply:
sorry, my fault. reverted back